components/data-table/row-actions.jsx

```diff
@@ -46,6 +46,10 @@ const DataTableRowActions = createReactClass({
 		 * `DataTable` row item
 		 */
 		item: PropTypes.object,
+		/**
+		 * Disable hint styling which changes the color of the dropdown svg on hover over.
```

Comment:
@interactivellama didn't want to worry the user about child/ancestor elements since it's all self contained within this component. The ancestor is the dropdown and the child is the icon.

Reply:
Does this automatically add to the doc site? I have no idea how this works.

Reply:
Yes, `npm run build-docs` will create `components/component-docs.json` which is ingested by the doc site.

Reply:
The ancestor is the table row. Hover over the row on the left side and watch the color change of the button on the right side.